CHANGELOG.md

```diff
@@ -2,6 +2,106 @@
 
 ## [Unreleased]
 
+## [1.8.0] - 2022-12-14
+
+Happy holidays! This release includes two big items we're pretty excited about and want to call out explicitly:
+
+* CPU Profiling 2.0 is now in beta!
+* `redis` 5 instrumentation upgrade notes
+
+### CPU Profiling 2.0 is now in beta!
```

Comment:
Maybe that's me but I kind of expected `CHANGELOG.md` to be very synthetic, and if a significant amount of text is required it would be present in other relevant documents and linked to from here. Maybe a file in `docs`? WDYT?

Reply:
I hear your concern:

* I hesitate a bit about adding it to a doc since this is somewhat tied to 1.8.0 (e.g. we may want to update it for 1.9.0).
* So instead I just removed the detailed text and linked back to the release notes, which will also contain the same text.

Hopefully this is reasonable :)

Reply:
Makes sense!